pkg/crt/runtimes.go

```diff
@@ -106,11 +113,12 @@ func AvailableRuntimes() []string {
 
 func AutoSelectRuntime() string {
 	available := AvailableRuntimes()
+	log.Debugf("Available runtimes: %v", available)
 	if len(available) > 0 {
 		return available[0]
 	}
 
-	return DockerRuntime
+	return DockerRuntime // Question -> This runtime may not necessarily be available?
```

Comment:
Yes, it is a good idea to check if the default/assumed runtime (when no explicit runtime param is specified) is available. A wishlist enhancement :-)

Reply:
A future TODO